.github/workflows/rust.yaml

```diff
@@ -21,18 +18,33 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: 'recursive'
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            rust:
+              - 'plugin/**'
+              - '**.nix'
+              - 'Cargo.toml'
```

Comment:
Will we add Cargo.lock to root? :)

Reply:
Not sure, but this will work regardless :)